geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json/JsonSimpleFeatureConverter.scala

```diff
@@ -112,6 +112,7 @@ object JsonField {
             jsonType: String,
             pathIsRoot: Boolean): BaseJsonField[_] = jsonType match {
     case "string"           => StringJsonField(name, expression, jsonConfig, transform, pathIsRoot)
+    case "date"             => StringJsonField(name, expression, jsonConfig, transform, pathIsRoot)
```

Comment:
we probably don't want to have a `date` type unless we're going to convert it to a Date for them...